docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,71 @@
+
+For further information regarding pro commands, refer to our :ref:`Getting started-with Ubuntu Pro client tutorial<tutorial-commands>`.
+
+Monitoring active machines:
+---------------------------
+
+Active machines are counted based on the number of machines attached to your subscription at a given time.
+
+- Machine Limits: 
+    Each subscription type has a limit on the number of machines it can cover. For personal subscriptions, this limit is 5 physical machines with unlimited VMs on these machines. For enterprise subscriptions, the limit is based on the number of physical hosts and VMs. 
+    For extensive use beyond 5 machines, consider an enterprise subscription.
+
+- Active Machine Tracking: 
+    The Ubuntu Pro dashboard and tools like `Landscape <Landscape_>`_ help track the number of active machines, ensuring compliance with subscription limits.
+
+- Unused Licenses and Active Users:
+    The Ubuntu Pro dashboard provides a comprehensive view of all active and attached machines. It shows the number of active machines and helps track unused Landscape licenses, ensuring that you remain within the subscription limits.
+
+- Monitoring with Landscape:
+    Setup: Install the Landscape client and connect it to the Landscape server. For instructions on how to install the Landscape client, refer to the `Landscape Installation Guide <Landscape_install_>`_.
+
+    Audit Machines: Use Landscape’s remote script execution capability to run audit scripts, such as ``uastatus.sh``, to gather detailed information about your Ubuntu Pro subscription across all machines. This information can be exported into a CSV report for further analysis.
+
+Dealing with multiple VMs:
+---------------------------
+
+- VM License Management: 
+    In an IaaS environment where node control is limited, you can contact `Canonical's sales team <pro_sales_form_>`_ to discuss purchasing licenses per single VM rather than per physical node.
+
+- Centralised Management: 
+    For enterprise environments, using Landscape ensures all Ubuntu instances have the necessary Ubuntu Pro entitlements, helping identify any discrepancies or unauthorised usage.
+
+Handling overage of active machines:
+------------------------------------
+
+Canonical monitors the number of active machines. Occasional overages are generally acceptable, but persistent overages may lead to subscription termination.
+
+Frequently Asked Questions:
+===========================
+
+- Can I use Ubuntu Pro Desktop for server use cases?
+    No, using Ubuntu Pro Desktop for non-desktop use cases violates the terms of service. Use an Ubuntu Pro Server subscription for server use.
+
+- How can I handle Ubuntu Pro in airgapped environments?
+    Use an airgapped contract server to distribute tokens within isolated environments, coordinated with Canonical’s Customer Success team. Using Ubuntu Pro in airgapped environments requires a paid subscription and is not available for free subscriptions.
```

Comment:
None of this seems particularly relevant to Pro tokens and machine activity.
```suggestion
```

Reply:
I was debating these being there too since we already point users to the FAQ page. This doc can do without these.